code/workspaces/web-app/src/components/common/form/AutocompleteTextSearch.js

```diff
@@ -154,6 +154,7 @@ function renderSuggestion({ suggestion, index, itemProps, highlightedIndex, sele
 }
 
 function getSuggestions(searchValue, suggestions) {
+  console.log('GET SUGGESTIONS', suggestions, searchValue);
```

Comment:
Remove this?

Reply:
👍 I'll remove in the next PR with more test migrations.